core/src/main/java/fj/Ord.java

```diff
@@ -428,21 +318,13 @@ public Ordering f(final Option<A> o2) {
    * @return An order instance for the {@link Either} type.
    */
   public static <A, B> Ord<Either<A, B>> eitherOrd(final Ord<A> oa, final Ord<B> ob) {
-    return new Ord<Either<A, B>>(new F<Either<A, B>, F<Either<A, B>, Ordering>>() {
-      public F<Either<A, B>, Ordering> f(final Either<A, B> e1) {
-        return new F<Either<A, B>, Ordering>() {
-          public Ordering f(final Either<A, B> e2) {
-            return e1.isLeft() ?
-                   e2.isLeft() ?
-                   oa.f.f(e1.left().value()).f(e2.left().value()) :
-                   Ordering.LT :
-                   e2.isLeft() ?
-                   Ordering.GT :
-                   ob.f.f(e1.right().value()).f(e2.right().value());
-          }
-        };
-      }
-    });
+    return new Ord<Either<A, B>>(e1 -> e2 -> e1.isLeft() ?
```

Comment:
Using `Ord.ord` starts to really be beneficial when the type becomes more complex.